package.json

```diff
@@ -80,7 +81,7 @@
     "workshopper-adventure-test": "^1.1.2"
   },
   "scripts": {
-    "lint": "./node_modules/.bin/standard",
+    "lint": "./node_modules/.bin/standard --fix",
```

Comment:
This should not be part of THIS pr! (Note: during development you can run `npm run lint -- --fix`